integtest/iceberg_real_hsi_test.py

```diff
@@ -39,15 +39,18 @@
 # Determine if the conditions are right for these tests
 we_are_running_on_an_iceberg_computer=False
 hostname=os.uname().nodename
-#if "iceberg01" in hostname or "protodune-daq02" in hostname:
 if "iceberg01" in hostname:
     we_are_running_on_an_iceberg_computer=True
-the_global_timing_partition_is_running=False
-username=os.environ.get('USER')
-for proc in psutil.process_iter(['pid', 'name', 'username']):
-    if proc.username() == username and "nanotimingrc" in proc.name():
-        the_global_timing_partition_is_running=True
-print(f"DEBUG: hostname is {hostname}, iceberg-computer flag is {we_are_running_on_an_iceberg_computer} and global-timing-running flag is {the_global_timing_partition_is_running}.")
+the_global_timing_session_is_running=False
+for proc in psutil.process_iter():
+    if "nanotimingrc" in proc.name() and "iceberg-integtest-timing-session" in proc.cmdline():
+        the_global_timing_session_is_running=True
+try:
+  urllib.request.urlopen('http://localhost:15879').status
+  the_connection_server_is_running=True
+except:
+  the_connection_server_is_running=False
+print(f"DEBUG: hostname is {hostname}, iceberg-computer flag is {we_are_running_on_an_iceberg_computer}, global-timing-running flag is {the_global_timing_session_is_running}, connection-server-running flag is {the_connection_server_is_running}.")
```

Comment:
Since you've made it independent of user, should this also print the username that the global timing session is running under?

Reply:
the_global_timing_session_username=nobody
then, at L47, the_global_timing_session_username = proc.username()